default/specials.txt

```diff
@@ -1931,5 +2011,23 @@ CHANCE_OF_GUARD_1
             effects = CreateShip designname = OneOf("SM_GUARD_0", "SM_GUARD_1")
 '''
 
+ANCIENT_RUINS_DEPLETED_CONTENT
+'''    stealth = 0
+    spawnrate = 0.0
+    effectsgroups = [
+        EffectsGroup
+            scope = Source
+            effects = SetTexture name = "planets/overlays/ruins.png"
+
+        EffectsGroup
+            scope = And [
+                Source
+                Focus type = "FOCUS_RESEARCH"
+            ]
+            effects = SetTargetResearch value = Value + Target.Population * 5 * [[RESEARCH_PER_POP]]
+        ]
+    graphic = "icons/specials_huge/ancient_ruins_excavated.png"
+'''
```

Comment:
I'm probably missing something, but after going over all the changes of this PR I can't find where this macro is being used... so what's the purpose of this new macro? Or is that a leftover of the first draft, where you had several versions of the ANCIENT_RUINS_DEPLETED_SPECIAL?